Category filter styling

gkalvatcheva opened the pull request:
• replaced some values with variables
• variables for icon size and font weight for subcategory filters
• width and height adjustments for open down vs open up panel
• removed entire .above section from expanding when we open the
subcategories panel

gkalvatcheva committed:
Merge branch 'master' into category-filter-styling

gkalvatcheva committed:
• category filter icon change
• category filter new variables
• metric sections big number size and icon size variables
• style for filter subcategory on hover, text, spacing, list styling
To do: clean up by using variables and not values

gkalvatcheva committed:
• icon charts in the first charts tabs - replaced background icon with
image
• new variables for icon image spacing and width
• adjusted position for expanded subcategory filters
• removed background color in the flyout
• update in hover background color

gkalvatcheva committed:
• new dimension variables for subcategory filters
• replaced some values with variables
• variables for icon size and font weight for subcategory filters
• width and height adjustments for open down vs open up panel
• removed entire .above section from expanding when we open the
subcategories panel

gkalvatcheva committed:
Merge branch 'master' into category-filter-styling

Conflicts:
	app/scripts/components/dashboard/charts-container.jsx
	app/scripts/components/dashboard/sub-category-filter.jsx
	app/stylesheets/dashboard/category-filter.scss
	app/stylesheets/dashboard/charts-container.scss

gkalvatcheva committed:
- error cleanup

ddimunzio committed:
Lint is complaining about the ,

esoliani commented on app/scripts/components/dashboard/charts-container.jsx:
```diff
@@ -0,0 +1,58 @@
+import React, { PropTypes } from 'react';
```
have you done any change to this file. it should not be used.

esoliani replied:
Please remove this file, so we don't misunderstand again

gkalvatcheva committed:
Merge branch 'master' into category-filter-styling

esoliani commented on app/scripts/components/dashboard/sub-category-filter.jsx:
```diff
@@ -56,6 +56,7 @@ const SubCategoryFilter = React.createClass({
     return (
       <div className="sub-category-filter">
         <div className="category-collapsable" onClick={this.toggle}>
+          <T k={`charts.sub-category.${this.props.type}`} />&nbsp;<Icon type={`chevron-${direction}`}/>
```
this is duplicating the categories.wrong!

galinagk replied:
Mmm, I change the arrow icon only here in icon type.

esoliani replied:
maybe it's a merge issue

gkalvatcheva committed:
Merge remote-tracking branch 'origin/category-filter-styling' into category-filter-styling

gkalvatcheva committed:
• overview stats background update ( according to mockup)
• dimensions variable update
• removed double style from charts-container.scss
• toggle label spelling change

gkalvatcheva committed:
Updates in subcategory section
• subcategory icon update (we ended up with 2 icons)
• reduced subcat. items padding
• font style update to subcat. item

gkalvatcheva committed:
Styling improvements of metric stats
• changed line height of medium status icon to vertically align the
symbol
• new variable for good status symbol to fit into the container
• adjustments of position and spaces within metric stat

Commit:
remove unused files
